prac_05/wimbledon.py

```diff
@@ -0,0 +1,58 @@
+"""
+wimbledon
```

Comment:
Capital letter for proper noun 🤓🤓🤓🤓🤓🤓 (your code is good I am grasping at straws)